### DISCUSSION TITLE
PimCore

### DISCUSSION THREAD

**Discussion by arulselvan008** [pimcore/pimcore#13675] [General]
Hi, I am new to Pimcore development .

Which IDE does Pimcore's backend code editor use?

How do I build my development code and deploy a new server ?

Plz let me know .

**Comment by kingjia90** [comment]
In the core team, we use Phpstorm as IDE and to build dev code with Docker (If on Windows, also setup WSL2). 
Recommending to start with the [Demo](https://github.com/pimcore/demo) but for a project from scratch, there's the [skeleton](https://github.com/pimcore/skeleton)
